src/models/export/cipher.ts

```diff
@@ -42,6 +43,7 @@ export class Cipher {
         view.name = req.name;
         view.notes = req.notes;
         view.favorite = req.favorite;
+        view.reprompt = CipherRepromptType.None;
```

Comment:
Feels like we should be able to use the actual reprompt value here?
```suggestion
        view.reprompt = req.reprompt;
```